microsoft/electionguard-python
# #722 Join Decryption

### Issue

Fixes #722 

### Description
Guardians can now join decryptions.  When they do they use their private key to create a partial decryption which is stored in the database.  Once all guardians have provided their partial decryptions the admin automatically decrypts and stores the result back into the database.  Neither the tally nor the spoiled ballots are displayed yet, that will be implemented in a subsequent PR.

### Testing
1. Create an election
2. Create a decryption
3. Have guardians log in, they should see a new decryption like:

![image](https://user-images.githubusercontent.com/1769905/181600535-fad01909-dc71-4aea-b727-9266050d7397.png)

4. Have guardian click the decryption, their screen should look like this:

![image](https://user-images.githubusercontent.com/1769905/181600874-6f4ead42-7b10-4e9c-bfdc-1f4c0128235e.png)

5. Have all guardians click the join button.  

After the 1st guardian clicks the admin's screen should look like this:

![image](https://user-images.githubusercontent.com/1769905/181601222-03bca362-c830-4d30-b490-0c3e5c27947d.png)

After all guardians have joined the admin screen should look like this:

![image](https://user-images.githubusercontent.com/1769905/181601335-4ae033d4-886e-4329-b069-3176b3d4822f.png)
